pkgs/development/python-modules/sphinx-markdown-parser/default.nix

```diff
@@ -25,7 +24,7 @@ buildPythonPackage rec {
 
   src = fetchFromGitHub {
     owner = "clayrisser";
-    repo = "sphinx-markdown-parser";
+    repo = pname;
```

Comment:
We'd like to keep theses as strings, since they're semantically something different.